packages/base-styles/_variables.scss

```diff
@@ -4,15 +4,15 @@
 
 // Fonts & basics
 $default-font: -apple-system, BlinkMacSystemFont,"Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,"Helvetica Neue", sans-serif;
-$default-font-size: 13px;
-$default-line-height: 1.4;
+$default-font-size: 14px;
+$default-line-height: (20px / $default-font-size); // line-height in px divided by font-size
 $editor-font: "Noto Serif", serif;
 $editor-html-font: Menlo, Consolas, monaco, monospace;
 $editor-font-size: 16px;
 $default-block-margin: 28px; // This value provides a consistent, contiguous spacing between blocks (it's 2x $block-padding).
 $text-editor-font-size: 14px;
 $editor-line-height: 1.8;
-$big-font-size: 18px;
+$big-font-size: 16px;
```

Comment:
Do these typography related variables need to exist now that the mixins exist? Can they be removed or at least deprecated?

Reply:
Only the ones that are applied to the content of the post. The type scale shouldn't be applied there because it represents the front end of the site, which is not in the scope of the type scale. For now, the type scale only applies to the application UI.

Reply:
🆒🆒

Why did it change as part of this PR?

Reply:
I think `$big-font-size` may be used for the UI, but I'd have to check and see why I changed it.

Reply:
Looks like it's not used that much, but I think I changed it because of this class

.block-editor-inserter__menu-help-panel-title {

which is some sort of title in the inserter popover.

Since it's only used a couple times, it might be easier just to eliminate the need for `$big-font-size`